docker-compose.yml

```diff
@@ -33,4 +33,22 @@ services:
       - "80:80"
     logging:
       options:
-        max-size: "10m"
\ No newline at end of file
+        max-size: "10m"
+  
+  cvmanager_postgres:
+    image: postgis/postgis:latest
+    restart: always
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    volumes: 
+      - pgdb:/var/lib/postgresql/data
+      - ./documents/sql_scripts:/docker-entrypoint-initdb.d

```

Comment:
I'll have to have you walk me through how these scripts get executed - very cool!